pmm-app/tests/pages/pmmSettingsPage.js

```diff
@@ -6,6 +6,7 @@ const locateLabel = (dataQA) => locate(`[data-qa="${dataQA}"]`).find('span');
 module.exports = {
   url: 'graph/d/pmm-settings/pmm-settings',
   prometheusAlertUrl: '/prometheus/rules',
+  alertsRules: '/prometheus/alerts',
```

Comment:
please don't call it alertRules, because that's not the end point to check rule, but the current state of alerts.

Reply:
Will rename! :)